src/runtime/crdt/tests/crdt-collection-test.ts

```diff
@@ -222,3 +222,5 @@ describe('CRDTCollection', () => {
     assert.deepEqual(modelChange2, otherChange2);
   });
 });
+
+// TODO: if/when adding more tests to this file, please, also update CollectionDataTest.java
```

Comment:
Perhaps we should file a P2 issue to automate this -- we could add something to sigh that at least outputs a warning or something, when a file in a whitelist is modified in a PR, to remind the author to not forget to change the corresponding file. And vice versa with the Java side.

Reply:
i added an equivalent note to the java file. 
about the automation - this won't apply to only tests then, but also all the parallel implementation files, will it? that's quite a lot of files, and i also doubt that they'll be always changed in the same pr.. so at least until the java implementation is more mature, i'm not sure the automation will be really helpful.